src/adlmidi_private.hpp

```diff
@@ -724,9 +723,8 @@ class MIDIplay
         }
         void updateBendSensitivity()
         {
-            int cent = bendsense_msb + static_cast<int>(static_cast<double>(bendsense_lsb) * (1.0 / 128.0));
-            bendsense = static_cast<double>(cent) / 8192.0;
-            bend = bendSrc * bendsense;
+            int cent = bendsense_msb * 128 + bendsense_lsb;
+            bendsense = cent * (1.0 / (128 * 8192));
```

Comment:
Okay, THIS formula produces the detuned sound. Just now I have made demo of your formula and the old formula:
[PitchBend-8192-results.7z.zip](https://github.com/Wohlstand/libADLMIDI/files/2067529/PitchBend-8192-results.7z.zip)

Reply:
I'll test this against Fluisynth. What your previous formula has done is simply to cancel the effect of sensitivity LSB.

I uploaded a tool because I lack controller ability in MIDI stuff and it is an impractical situation for test. https://github.com/jpcima/midi-shell
And I will add some XG ability, to be able to control my synthesizer.

Reply:
I have an element of answer for you: Fluidsynth **ignores** sensitivity LSB

Reply:
It's difficult to find any open source synths which implement this LSB. In Code search, I have not been able to find just one. Maybe we should just drop implementing this LSB value.... but I think it's really supposed to be 1/100ths values.

I'll make you an entire archives of MIDIs I have using pitch LSB. (I have lots)

Reply:
Have MIDI files. This thing is long to run on collections and here's part of findings.
It's files which use non-0 pitch sense LSB. You have pitch sense events logged in text files at the root.

[midis1.tar.gz](https://github.com/Wohlstand/libADLMIDI/files/2067924/midis1.tar.gz)

Reply:
All MIDI affected I have in this computer [fullmidi.tar.gz](https://github.com/Wohlstand/libADLMIDI/files/2067963/fullmidi.tar.gz)

Reply:
Okay, just now I have tested my PlusDemo on SB-Live's synth, and the sound is same distorted as your demo. So, i'll re-tune my demos and then I'll test them on updated OPxMIDI's code...